dts/bindings/sensor/ti,ina226.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    TI INA226 Bidirectional Current and Power Monitor.
+    The <zephyr/dt-bindings/sensor/ina226.h> file should be included in the
+    DeviceTree and it provides macros that can be used for initializing the
+    configuration registers.
+
+    Example:
+      /* Board - mr_canhubk3. */
+      &lpi2c1 {
+        status = "okay";
+        pinctrl-0 = <&lpi2c1_default>;
+        pinctrl-names = "default";
+        clock-frequency = <I2C_BITRATE_STANDARD>;
+
+        ina226: ina226@41 {
+          compatible = "ti,ina226";
+          reg = <0x41>;
+          current-lsb-microamps = <5000>;
+          rshunt-micro-ohms = <500>;
+        };
+      };
+
+
+compatible: "ti,ina226"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  config:
+    type: int
+    default: 0x0000
+    description: |
+      Value of the configuration register.
+
+      It's a 16 bit value that contains the following fields:
+        1. Bits 2 - 0: "operating-mode" property.
+        2. Bits 5 - 3: "vshunt-conversion-time-us" property.
+        3. Bits 8 - 6: "vbus-conversion-time-us" property.
+        4. Bits 11 - 9: "avg-count" property.
+        5. Bits 14 - 11: reserved. (default = 0b001.)
+        6. Bit 15: reset bit (setting this bit 1 generates a system reset).
+
+      It's value is derived from the respective properties of fields at init.
+        Although any value that we assign to it will override the value set in
+        the respective properties of the individual fields because we take the
+        bitwise OR of it and the dependent properties at device init.
+
+      Default is 0x0000 because it's only meant to control bit 15 (reset bit).
+        All other components of this property are meant to be assigned by
+        setting their respective property. e.g. for setting bits 2 - 0, we
+        should set the "operating-mode" property.
+        Note: The power-on reset value of configuration register is 0x4127.
```

Comment:
Why do we need this property at all then? We shouldn't have redundant properties

Reply:
Yes that's correct. I have removed this property in the latest push. Sorry I didn't see it before.